lib/components/TooltipRenderer/TooltipRenderer.treat.ts

```diff
@@ -1,18 +1,22 @@
 import { style } from 'sku/treat';
 
+export const constants = {
+  maxWidth: 260,
+};
+
 export const background = style((theme) => ({
   background: theme.color.foreground.neutral,
 }));
 
 export const maxWidth = style({
-  maxWidth: 260,
+  maxWidth: constants.maxWidth,
 });
 
 export const verticalOffsetBeforeEntrance = style({
-  transform: 'translateZ(0) translateY(-4px)',
+  transform: 'translateZ(0) translateY(4px)',
   selectors: {
-    '[data-popper-placement*=top] &': {
-      transform: 'translateZ(0) translateY(4px)',
+    '[data-popper-placement*=bottom] &': {
+      transform: 'translateZ(0) translateY(-4px)',
```

Comment:
should the -4px value be a theme variable (grid?) or will it work across all themes at -4px?

Reply:
This transition isn't following the grid, it's just a coincidence :)